micro-rdk/src/common/conn/viam.rs

```diff
@@ -194,6 +194,33 @@ where
     <Storage as RobotConfigurationStorage>::Error: Debug,
     ServerError: From<<Storage as RobotConfigurationStorage>::Error>,
 {
+    pub fn get_default_max_concurrent_connections() -> usize {
+        // By default, we get three, everywhere.
+        #[allow(unused_mut)]
+        let mut max = 3;
+
+        // If local signaling is enabled, grant two more
+        #[cfg(feature = "local-signaling")]
+        {
+            max += 2;
+        }
+
+        // But on an esp32 lacking SPIRAM, assume only one connection can be realized
```

Comment:
What's the behavior of an SDK trying to connect if local signaling is enabled but lacking spiram (when max connections is 1)? 

might be good to have a log message or something to give the user a heads-up about the limitation.
`log::warn!("this device can only support one connection at a time blah blah blah");`

Reply:
Yes, actually, you've caught something that I intended to do, but then forgot to execute on. My intention was that if max conns is too small, to disable local signaling, with a warning.

I will add logic for that and post an update.

Reply:
@mattjperez - Fixed this in latest.